borgmatic/commands/arguments.py

```diff
@@ -629,6 +629,13 @@ def make_parsers():
         dest='databases',
         help="Names of databases to restore from archive, defaults to all databases. Note that any databases to restore must be defined in borgmatic's configuration",
     )
+    restore_group.add_argument(
+        '--schema',
+        metavar='NAME',
+        nargs='+',
+        dest='schemas',
+        help="Names of schemas to restore from the database, defaults to all schemas."
```

Comment:
Maybe worth noting "if supported by your database system" or whatever? Or even specifying the database hooks that support this?